### PR TITLE
Release MSAL Angular as 1.0.0 stable

### LINKED ISSUE #786: Angular Wrapper in sync with core MSAL 1.0.x
This is a placeholder to track all the "Angular" specific issues which simply need the wrapper to be upgraded to the latest msal js core version.

- supports latest Angular versions with rxjs support
- is Msal JS 1.0.X compatible - with access to the new API Surfaced 

All tickets raised asking for this support will be tagged with this issue going forward. This is to simplify the traction from our end.

In case if anyone has specific Angular related requirement that you believe is not covered with the plan above, please feel free to drop a comment on this issue.

### LINKED ISSUE #1056: Add linting to msal-angular
The `msal-angular` project is currently without any enforced linting rules, which would help code quality and readability.

AC:
1. The `msal-angular` project is configured with eslint rules.

### LINKED ISSUE #1573: When will @azure/msal-angular come out of beta?
## Library
- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [x] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Description

When is `@azure/msal-angular@1.x.x` expected to come out of beta? We would like to use it but there are concerns using a Beta package in an app that is in production. As we are upgrading both `msal` and `angular` it would be great to cleanup our application and remove code that we don't want to maintain.

### PR DESCRIPTION
**Pull request by jasonnutter**
TODO:

- [x] Remove beta tag
- [x] Update peer dep of msal to 1.3.0
- [x] Add lint command, fix lint errors 
- [x] Update samples to msal@1.3.0
- [x] Clean up backlog
- [x] Add ssoSilent
- [x] Update samples to @azure/msal-angular@1.0.0
- [x] Changelog
- [x] README updates

Post PR changes:

- [ ] Update external samples
- [ ] Update portal

### CODE REVIEW

**Review by pkanher617 — APPROVED** [review]
Congrats on reaching 1.0!!!

**Review by tnorling — APPROVED** [review]

File: lib/msal-angular/changelog.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 1.0.0
+
+Stable release of MSAL Angular v1. See beta versions below for complete list of changes.
+
+Highlights:
+
+* Requires `msal@1.3.0`.
```
Comment (DarylThayil) [inline lib/msal-angular/changelog.md:9]: is this carrot?

**Review by DarylThayil — APPROVED** [review]

File: lib/msal-angular/changelog.md (hunk 1 above)
Comment (jasonnutter) [reply lib/msal-angular/changelog.md:9]: Yes, technically.

**Comment by sameerag** [comment]
Congratulations @jasonnutter !! This is a great milestone!!

**Review by sameerag — APPROVED** [review]

**Comment by rrgahan** [comment]
Does this address #598?

**Comment by jasonnutter** [comment]
@rrgahan Yes, the Guard will prompt for interaction fs the session has expired.

**Pull request merged**